backend/README.md

```diff
@@ -0,0 +1,49 @@
+## Run locally
+
+You'll need access to the Firebase project at https://console.firebase.google.com/project/c19-backend.
+
+### Sign into Firebase
+
+From the `backend` directory:
+
+```
+npm install -g firebase-tools
+firebase login
+```
+
+### Start the functions
+
+```
+cd functions
+yarn install
+firebase functions:config:get > .runtimeconfig.json
```

Comment:
When I run this command, I see the following contents in my `.runtimeconfig.json`

"Error: HTTP Error: 403, The caller does not have permission"

So it looks like I'm still having some permissions issues even though I was able to login with `firebase login`

Reply:
You shouldn't get this error any more now that you're and admin on the project.